superchain/configs/mainnet/worldchain.toml

```diff
@@ -20,7 +20,7 @@ max_sequencer_drift = 1800
   holocene_time = 1738238400 # Thu 30 Jan 2025 12:00:00 UTC
 
 [optimism]
-  eip1559_elasticity = 2
```

Comment:
These fields should be in the genesis part of the config. There are also some roles that should be in the genesis part of the config. We should have something dynamic that fetches the values from onchain if we care about what the values are at the moment.